bin/faucet/src/server/resources/index.js

```diff
@@ -229,7 +241,6 @@ document.addEventListener('DOMContentLoaded', function () {
                 // Take the first 8 bytes of the hash and parse them as u64 in big-endian
                 let digest = BigInt("0x" + hash.hex().slice(0, 16));
 
-                // Check if the hash is less than the target
```

Comment:
I would leave these comments, and maybe even expand them to make it more comprehensive